app/src/main/res/drawable/ic_share_blue_24dp.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:alpha="0.84" android:height="@dimen/half_standard_height"
+  android:viewportHeight="24.0" android:viewportWidth="24.0"
+  android:width="@dimen/half_standard_height" xmlns:android="http://schemas.android.com/apk/res/android">
+  <path android:fillColor="#0C609C" android:pathData="M18,16.08c-0.76,0 -1.44,0.3 -1.96,0.77L8.91,12.7c0.05,-0.23 0.09,-0.46 0.09,-0.7s-0.04,-0.47 -0.09,-0.7l7.05,-4.11c0.54,0.5 1.25,0.81 2.04,0.81 1.66,0 3,-1.34 3,-3s-1.34,-3 -3,-3 -3,1.34 -3,3c0,0.24 0.04,0.47 0.09,0.7L8.04,9.81C7.5,9.31 6.79,9 6,9c-1.66,0 -3,1.34 -3,3s1.34,3 3,3c0.79,0 1.5,-0.31 2.04,-0.81l7.12,4.16c-0.05,0.21 -0.08,0.43 -0.08,0.65 0,1.61 1.31,2.92 2.92,2.92 1.61,0 2.92,-1.31 2.92,-2.92s-1.31,-2.92 -2.92,-2.92z"/>
```

Comment:
Instead of creating two separate vector drawables, we can just change the fillColor based on the theme

Reply:
Sorry about the unnecessary changes. I will try to fix it with less changes to the project.